cg/apps/demultiplex/sbatch.py

```diff
@@ -16,9 +16,9 @@
 touch {demux_completed_file}"
 
 singularity exec --bind \
-/home/proj/{environment}/sequencing_data/illumina_novaseq_flow_cells/demultiplexed-runs,\
-/home/proj/{environment}/sequencing_data/illumina_novaseq_flow_cells/flow_cells,\
-/home/proj/{environment}/sequencing_data/illumina_novaseq_flow_cells/flow_cells/'$SLURM_JOB_ID':/run/user/$(id -u) \
+/home/proj/{environment}/sequencing_data/illumina/demultiplexed-runs,\
+/home/proj/{environment}/sequencing_data/illumina/low_cells,\
```

Comment:
Should be flow_cells right?
```suggestion
/home/proj/{environment}/sequencing_data/illumina/flow_cells,\
```

Reply:
Nice catch. Would be real bad I messed this up one more time 😅